walkthroughs/eks-getting-started/infrastructure/yelb_initial_deployment.yaml

```diff
@@ -88,7 +88,7 @@ spec:
     spec:
       containers:
         - name: yelb-ui
-          image: mreferre/yelb-ui:0.6
+          image: mreferre/yelb-ui:0.7
```

Comment:
What's the difference between these two versions?

Reply:
It was a change so the application can support https. Before it, http was hardcoded so all of the api calls to get stats and votes were being done over http.

Reply:
Please refer to comment above